CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+## Contributer's Guide
+
+Thanks for your interest in the project! We welcome pull requests from
+developers of all skill levels. To get started, simply fork the master branch
+on GitHub to your personal account and then clone the fork into your
+development environment.
+
+Padmakar Ojha (**dvopsway** on github) is the original creator of the
+SSHmux projet, and currently maintains the project along with Rob Haverkamp (**rjrhaverkamp** on GitHub and Twitter).
+
+Please don't hesitate to reach out if you have any questions, or just need a
+little help getting started.
+
+### Pull Requests
+
+Before submitting a pull request, please ensure you have added or updated tests as appropriate, and that all existing tests still pass with your changes on both Python 2 and Python 3. Please also ensure that your coding style follows PEP 8.
+
+
+### Test coverage
+
+Pull requests won't be merged when test coverage decreases. This helps ensure the quality of the project. To check coverage before submitting a pull request:
+
+```bash
+$ COVERAGE_FILE=.coverage_ssh coverage run tests/test_ssh.py
+$ COVERAGE_FILE=.coverage_validations coverage run tests/test_validations.py
+$ coverage combine .coverage_ssh .coverage_validations
+```
+
+### Code style rules
```

Comment:
Do we need this whole section, Looks like a lot. A simple flake8 check will ensure if everything is good or not.